lsp/nls/src/linearization/mod.rs

```diff
@@ -393,6 +395,37 @@ impl<'a> Linearizer for AnalysisHost<'a> {
                     })
                 }
             }
+            Term::ResolvedImport(file) => {
+                fn final_term_pos(term: &RichTerm) -> &TermPos {
+                    let RichTerm { term, pos } = term;
+                    match term.as_ref() {
+                        Term::Let(_, _, body, _) | Term::LetPattern(_, _, _, body) => {
+                            final_term_pos(body)
+                        }
+                        Term::Op1(UnaryOp::StaticAccess(field), _) => &field.pos,
+                        _ => pos,
+                    }
+                }
+
+                let Some(linearization) = lin.lin_cache.get(file) else {
+                    return
+                };
+                // This is safe because the import file is resolved, before we linearize
+                // the containing file, therefore the cache MUST have the term stored.
+                let term = lin.cache.get(*file).unwrap();
+                let position = final_term_pos(&term);
+                let locator = (*file, position.unwrap().start);
```

Comment:
is it the case that `position` is always defined? If yes, maybe add a comment to explain why as well. If not, maybe unwrapping isn't the best idea, and we should just return as well.